File: distros/J/JavonetPerlSdk/lib/Javonet/Binaries/Python/javonet/core/generator/handler/shared/body/SharedBodyHandler.py
from javonet.core.generator.handler.AbstractGeneretatorHandler import AbstractGeneratorHandler


class SharedBodyHandler(AbstractGeneratorHandler):
    def generate_command(self, analyzed_object, parent_command, handlers):
        return analyzed_object

    def generate_code(self, existing_string_builder, common_command, used_object, handlers):
        handlers.SHARED_BODY_HANDLER[used_object.command_type].generate_code(existing_string_builder, common_command,
                                                                             used_object, handlers)

File: distros/J/JavonetPerlSdk/lib/Javonet/Binaries/Nodejs/core/generator/handler/shared/body/SharedBodyHandler.js
const AbstractGeneratorHandler = require("../../AbstractGeneratorHandler");
const os = require('os')

class SharedBodyHandler extends AbstractGeneratorHandler {

    generate_code(existing_string_builder, common_command, used_object, handlers) {
        existing_string_builder.append("{");
        existing_string_builder.append(os.EOL);
        handlers.SHARED_BODY_HANDLER[used_object.commandType].generate_code(existing_string_builder, common_command, used_object, handlers)
        existing_string_builder.append("}");
    }
}

module.exports = SharedBodyHandler